.dev/docker/env/Dockerfile

```diff
@@ -23,19 +23,18 @@ RUN apt-get update \
       clang \
       wget
 
-RUN if [ "$USE_CUDA" = "true" ] ; then wget -qO- https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz | tar xz \
-     && apt install -y libblas-dev \
-     && cd eigen-3.3.7 \
-     && mkdir build \
-     && cd build \
-     && cmake .. \
-     && make install \
-     && cd ../.. \
-     && rm -rf eigen-3.3.7/ \
-     && rm -f eigen-3.3.7.tar.gz ; \
-    fi
+RUN wget -qO- https://gitlab.com/libeigen/eigen/-/archive/3.3.7/eigen-3.3.7.tar.gz | tar xz \
```

Comment:
I know CUDA isn't ready for 20.04 yet, but I'd like there to be a CI with default Eigen version too, for making sure we don't break dependencies unknowingly.

My comment wrt Eigen was that you were installing eigen (and still are) in 2 ways: manually and via apt

Reply:
> I know CUDA isn't ready for 20.04 yet, but I'd like there to be a CI with default Eigen version too, for making sure we don't break dependencies unknowingly.

I am not quite sure what you mean.Do you meant a separate pipeline for the default version or just use the one from apt in Ubuntu 20?

> My comment wrt Eigen was that you were installing eigen (and still are) in 2 ways: manually and via apt
Thanks! DIdn't notice this, will remove the one from apt.

Reply:
Eigen 3.3.7 will have bug fixes more than just for CUDA. We need to make sure we have a lower Eigen version compilation (without CUDA) to prevent accidental non-support

Reply:
Agreed, So then we can have separate pipeline (without CUDA) in the baseline Ubuntu version and a separate pipeline for CUDA. Did you have something else in mind?

Reply:
No, you pretty much nailed it.

Reply:
Didn't remove default Eigen from apt since it will still be used on 16.04. Currently, it seems simpler for both version Eigen to exist in the image and we use the one we want.